dialects/linux/dlsof.h

```diff
@@ -40,7 +40,6 @@
 
 #include <dirent.h>
 #define	DIRTYPE	dirent			/* for arg.c's enter_dir() */
-#define	__USE_GNU			/* to get all O_* symbols in fcntl.h */
```

Comment:
This is required for `case-20-open-flags-path` to work.

Reply:
Thanks, I'll add a #ifndef __USE_GNU before this.